node/src/accountant/payable_dao.rs

```diff
@@ -635,63 +855,68 @@ mod tests {
 
     fn transaction_confirmed_works(
         test_name: &str,
-        (initial_amount, balance_change, expected_balance_after): (i128, u128, u128),
+        (initial_amount_1, balance_change_1, expected_balance_after_1): (u128, u128, u128),
     ) {
         let home_dir = ensure_node_home_directory_exists("payable_dao", test_name);
+        //a hardcoded set that just makes a complement to the crucial, supplied one; this points to the ability of
+        //handling multiple transactions together
+        let initial_amount_2 = 5_678_901;
+        let balance_change_2 = 678_902;
+        let expected_balance_after_2 = 4_999_999;
         let boxed_conn = DbInitializerReal::default()
             .initialize(&home_dir, DbInitializationConfig::test_default())
             .unwrap();
-        let hash = H256::from_uint(&U256::from(12345));
-        let rowid = 789;
-        let previous_timestamp = from_time_t(190_000_000);
-        let payable_timestamp = from_time_t(199_000_000);
-        let attempt = 5;
-        let wallet = make_wallet("bobble");
-        {
-            insert_record_fn(
-                &*boxed_conn,
-                &wallet.to_string(),
-                initial_amount,
-                to_time_t(previous_timestamp),
-                Some(sign_conversion::<u64, i64>(rowid).unwrap()),
-            );
-        }
+        let setup_holder = make_fingerprint_pair_and_insert_initial_payable_records(
+            boxed_conn.as_ref(),
+            initial_amount_1,
+            initial_amount_2,
+            balance_change_1,
+            balance_change_2,
+        );
         let subject = PayableDaoReal::new(boxed_conn);
-        let pending_payable_fingerprint = PendingPayableFingerprint {
-            rowid_opt: Some(rowid),
-            timestamp: payable_timestamp,
-            hash,
-            attempt_opt: Some(attempt),
-            amount: balance_change,
-            process_error: None,
+        let expected_status_before_1 = PayableAccount {
```

Comment:
This might be clearer if you moved the expected values into the Assert section, and moved the `.clone()`s accordingly.